### ISSUE TITLE
Error after deploying

### ISSUE THREAD

**Issue by pietroalbini** [rust-lang/crates-io-ops-bot#41]
When trying to deploy a version, after the build finished the bot posted this error:

```
There was an error when running deploy_app: CommandError("error decoding response body: invalid type: string "scheduler:standard", expected i64 at line 1 column 41")
```

**Comment by nellshamrell** [comment]
Just tested this out using staging - I believe the build and deploy are happening fine, but the bot is choking on the response from the Heroku API. Looking into this further.

**Comment by nellshamrell** [comment]
Fixed in #47
